.github/workflows/SDKSuppression-Label.yaml

```diff
@@ -0,0 +1,33 @@
+name: breakingchange suppression label Workflow
+
+on:
+  pull_request:
+    branches:
+      - jacktn/suppression-label-script 
+    types:
+      - opened
+      - synchronize
+      - reopened
+
+  workflow_dispatch:
+
+jobs:
+  pr_details:
```

Comment:
Can you rename 'pr_details' to something more descriptive, like `manage_sdk_suppression_labels`?